runners/samza/src/main/java/org/apache/beam/runners/samza/SamzaRunner.java

```diff
@@ -130,21 +130,23 @@ public SamzaPipelineResult run(Pipeline pipeline) {
           PipelineDotRenderer.toDotString(pipeline));
       LOG.debug(
           "Pre-processed Beam pipeline in json format:\n{}",
-          PipelineJsonRenderer.toJsonString(pipeline));
+          PipelineJsonRenderer.toJsonString(pipeline, Collections.emptyMap()));
     }
 
     pipeline.replaceAll(SamzaTransformOverrides.getDefaultOverrides());
 
+    final Map<PValue, String> idMap = PViewToIdMapper.buildIdMap(pipeline);
+    final Set<String> nonUniqueStateIds = StateIdParser.scan(pipeline);
+    final Map<String, Map.Entry<String, String>> transformIOMap =
```

Comment:
Let's not create this map here. Instead, create a ConfigContext and pass it in below.